src/docs/admin/ProActiveAdminGuide.adoc

```diff
@@ -1287,6 +1287,22 @@ The following important rules apply to global variables and generic information:
             </genericInformation>
 ----
 
+=== Manage job labels
+
+A label is a property that can be attached to a job.
+The advantage of having labels is given by the possibility to group and manipulate a certain number of jobs by filtering based on this "custom status" or label.
+For example, a company can require that jobs which are having errors being reviewed by operators and mark them as "reviewed" when completed. Job labels can be used for this purpose.
+
+Administrators can define which labels can be applied to jobs in the Admin menu of the Scheduler Portal (link to glossary). Physically, possible labels are stored in the scheduler database.
+
+The constraints of a label name can be configured in the `PROACTIVE_HOME/config/scheduler/settings.ini` file.
+
+The maximum length is specified by the property pa.scheduler.label.max.length.
+If this property is not specified, the default value is ``20``.
+
+The REGEX constrain is specified by the property pa.scheduler.label.regex.
```

Comment:
The regular expression constraint. Put also the properties between back quotes.
`pa.scheduler.label.max.length` `property pa.scheduler.label.regex`